Sources/UI/Channels/ChannelTableViewCell.swift

```diff
@@ -10,10 +10,21 @@ import UIKit
 import RxSwift
 
 /// Channel table view cell.
-public final class ChannelTableViewCell: UITableViewCell, Reusable {
+open class ChannelTableViewCell: UITableViewCell, Reusable {
+    
+    /// A channel style.
+    public var style: ChannelViewStyle = ChannelViewStyle()
+    /// Checks if needds setup layout.
+    public private(set) var needsToSetup = true
     
     /// An avatar view.
-    public let avatarView: AvatarView = AvatarView(cornerRadius: 20)
+    public private(set) lazy var avatarView: AvatarView = {
+        if let avatarStyle = style.avatarViewStyle {
```

Comment:
Why not just `return AvatarView(cornerRadius: style.avatarViewStyle?.radius ?? .channelAvatarRadius)`?

Reply:
Yes. I'll fix